templates/components/itilobject/service_levels.html.twig

```diff
@@ -215,6 +217,14 @@
                            myCollapse.show();
                         }
                      </script>
+
+                     {% if item.fields[la_field.lafieldname] != 0 %}
+                        <script type="text/javascript">
+                           $(document).ready(function() {
+                              toggleAssignLA_{{ rand }}();
+                           });
+                        </script>
+                     {% endif %}
```

Comment:
Are you sure of this ? Even when the SLA is already defined, I guess its value should still be updatable.

Reply:
It's always possible to modify them, in fact the SLA form is in datetime format by default, but what I've done is to change it to dropdown format on the SLA objects etc., like the basic behaviour that allows you to switch to dropdown when you press a button.